composer/cicd_sample/test-dags.cloudbuild.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START composer_cicd_unittest_yaml]
+
+steps:
+  # install dependencies
+  - name: python:3.8-slim
+    entrypoint: pip
+    args: ["install", "-r", "requirements.txt", "-c", "constraints.txt", "--user"]
+  
+  - name: python:3.8-slim
+    entrypoint: pip
+    args: ["install", "-r", "requirements-test.txt", "--user"]
+
+  # run in python 3.8 which is latest version in Cloud Composer
+  # SHORT_SHA is a default Cloud Build substitution
+  # that is the first seven characters of COMMIT_SHA
+  - name: python:3.8-slim
+    entrypoint: python3.8
+    args: ["-m", "pytest", "-s", "dags/", "--junitxml=${SHORT_SHA}_test_log.xml"]
```

Comment:
Are you doing anything with this test output?  Do you want to store it in a bucket or are you happy to just have it show up in the logs?

Reply:
I am not. I'm happy to just have it show up in the logs. I copied this xml bit from an example - is it necessary, or should I just remove it?

Reply:
I would remove the `--junitxml` flag -- in the sample, we're uploading those test logs to storage.  If you don't need that, then there is no reason to write the logs to a file!